sos/plugins/sosreport.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2019 Red Hat, Inc., Pavel Moravec <pmoravec@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin, UbuntuPlugin, DebianPlugin
+
+
+class Sosreport(Plugin, RedHatPlugin, UbuntuPlugin, DebianPlugin):
+    """Sos plugin collects information about SoS report itself
+    """
+
+    plugin_name = 'sos'
+
+    def setup(self):
+        self.add_copy_spec([
+            '/etc/sos.conf',
+            '/etc/sos.extras.d'
+        ])
```

Comment:
I would prefer that this was bound more closely to the thing that uses it - i.e. if that's the config source for an `extras` plugin, then let's collect it in that plugin - otherwise you're just asking for a user to run `extras`, while disabling `sosreport`.. and then we can't see what they did.

Reply:
I wanted to collect also `sos.conf` that does not belong to any plugin. But since `sos.log` contains line like `loaded options from config file: --batch --build --plugopts postgresql.dbport=54321,process.lsof=off'`, I think we dont need the file. So worth collecting the `extras` dir in its plugin.